README.md

```diff
@@ -1,12 +1,14 @@
 [![Build Status](https://travis-ci.org/OCA/purchase-workflow.svg?branch=7.0)](https://travis-ci.org/OCA/purchase-workflow)
```

Comment:
Well, it's another discussion. I am compliant with the template:
https://raw.githubusercontent.com/OCA/maintainer-quality-tools/master/sample_files/README.md